react-ui/src/pages/contest/ContestAppBarMain.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * Contest-specific content for top app bar
+ */
+
+import Box from '@material-ui/core/Box';
+import PropTypes from 'prop-types';
+import { useLocation, useParams } from 'react-router-dom';
+
+import {
+  useSwrData,
+} from '../../common';
+import {
+  ArrowBackButton,
+  Countdown,
+} from '../../components';
+
+export default function ContestAppBarMain({ handleVotingExpired, handleReload }) {
```

Comment:
Nice! I think when these used to be arrow functions, I needed to export them after defining the prop types, but this is much cleaner.